From 35d8a973fe4d38afee944db636c3d2b1df3741a7 Mon Sep 17 00:00:00 2001
Message-Id: <35d8a973fe4d38afee944db636c3d2b1df3741a7.1651196606.git.geliang.tang@suse.com>
From: "David E. Box" <david.e.box@linux.intel.com>
Date: Tue, 26 Jan 2021 12:55:06 -0800
Subject: [PATCH] platform/x86: intel_pmt: Make INTEL_PMT_CLASS
 non-user-selectable
Git-commit: 35d8a973fe4d38afee944db636c3d2b1df3741a7
Patch-mainline: v5.12-rc1
References: bsn#210
Modified-by-SEL: No

Fix error in Kconfig that exposed INTEL_PMT_CLASS as a user selectable
option. It is already selected by INTEL_PMT_TELEMETRY and
INTEL_PMT_CRASHLOG which are user selectable.

Fixes: e2729113ce66 ("platform/x86: Intel PMT class driver")
Signed-off-by: David E. Box <david.e.box@linux.intel.com>
Link: https://lore.kernel.org/r/20210126205508.30907-1-david.e.box@linux.intel.com
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/platform/x86/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/x86/Kconfig b/drivers/platform/x86/Kconfig
index a6290c32cfda..64131ef44747 100644
--- a/drivers/platform/x86/Kconfig
+++ b/drivers/platform/x86/Kconfig
@@ -1355,7 +1355,7 @@ config INTEL_PMC_CORE
 		- MPHY/PLL gating status (Sunrisepoint PCH only)
 
 config INTEL_PMT_CLASS
-	tristate "Intel Platform Monitoring Technology (PMT) Class driver"
+	tristate
 	help
 	  The Intel Platform Monitoring Technology (PMT) class driver provides
 	  the basic sysfs interface and file hierarchy uses by PMT devices.
-- 
2.34.1

